scripts/web_reports.py

```diff
@@ -31,7 +31,26 @@ def get_participation_string(participation, language):
         labels = labels,
         percentage = 0 if invited == 0 else (answered / invited * 100))
 
-def web_report_course(summary_path, stat_path, output_path, html_templates, courses):
+def create_chart_js(question, question_stats, scales, chart_id):
+    chart_data = []
+    answers = scales[question]["order"]
+    for answer in answers:
+        if answer in question_stats["counts"]:
+            count = question_stats["counts"][answer]
+        else:
+            count = 0
+        chart_data.append('{{ label: "{}", value: {} }}'.format(answer, count))
+    if len(answers) == 6:
+        colors = "['#1a9850', '#91cf60', '#d9ef8b', '#fee08b', '#fc8d59', '#d73027']"
+    elif len(answers) == 5:
+        colors = "['#d7191c', '#fdae61', '#91cf60', '#abd9e9', '#2c7bb6']"
+    else:
+        print("Warning: Chart for '{}' omitted. No colors defined for questions with {} alternatives".format(
```

Comment:
Kanskje legge til farger for flere varianter, 4 og 7 for eksempel? Spesielt siden vi kommer til å gjøre endringer i nettskjema snart.